### PR TITLE
Fix chainscore methods

### PR DESCRIPTION
**Pull request by inwonkim**
* validate integer param range
* revert min version of penalizeNonvoters

### CODE REVIEW

File: icon/iiss/icstate/networkvalue.go [hunk 1]
```diff
@@ -329,8 +329,8 @@ func (s *State) GetConsistentValidationPenaltySlashRatio() int {
 }
 
 func (s *State) SetConsistentValidationPenaltySlashRatio(value int) error {
-	if value < 0 {
-		return errors.IllegalArgumentError.New("ConsistentValidationPenaltySlashRatio must have positive value")
+	if value < 0 || value >= 100 {
```
Comment (mksong-icon) [inline icon/iiss/icstate/networkvalue.go:332]: Why 100% is not allowed?

File: icon/iiss/icstate/networkvalue.go [hunk 2]
```diff
@@ -349,7 +349,7 @@ func (s *State) GetNonvotedPenaltySlashRatio() int {
 }
 
 func (s *State) SetNonvotedPenaltySlashRatio(value int) error {
-	if value <= 0 {
+	if value < 0 || value >= 100 {
```
Comment (mksong-icon) [inline icon/iiss/icstate/networkvalue.go:352]: Why 100% is not allowed?

**Review by mksong-icon — COMMENTED** [review]
If it's allowed to set it as 100%, it should be allowed even though it's not allowed by policy.

File: icon/iiss/icstate/networkvalue.go [hunk 3]
```diff
@@ -378,7 +378,7 @@ func (s *State) GetNetworkInfoInJSON() (map[string]interface{}, error) {
 	jso["consistentValidationPenaltySlashRatio"] = s.GetConsistentValidationPenaltySlashRatio()
 	jso["unstakeSlotMax"] = s.GetUnstakeSlotMax()
 	jso["delegationSlotMax"] = s.GetDelegationSlotMax()
-	jso["proposalNonVotePenaltySlashRatio"] = s.GetNonvotedPenaltySlashRatio()
+	jso["proposalNonVotePenaltySlashRatio"] = s.GetNonVotedPenaltySlashRatio()
```
Comment (sink772) [inline icon/iiss/icstate/networkvalue.go:381]: Which one is correct?
proposalNonVotePenaltySlashRatio vs proposalNonVotedPenaltySlashRatio

**Review by mksong-icon — APPROVED** [review]

**Comment by mksong-icon** [comment]
Followings are also found

```
	VarNonvotedPenaltySlashRatio             = "nonvoted_penalty_slashRatio"
```

If it's not used by Rev16, you may change the value of the constant.

**Comment by inwonkim** [comment]
> Followings are also found
> 
> ```
> 	VarNonvotedPenaltySlashRatio             = "nonvoted_penalty_slashRatio"
> ```
> 
> If it's not used by Rev16, you may change the value of the constant.

@mksong-icon 
it's not used yet and i change the variable and its value

**Review by sink772 — APPROVED** [review]

**Review by mksong-icon — APPROVED** [review]

**Pull request merged**